forest/jasmine/traj2stats.py

```diff
@@ -732,182 +1379,126 @@ def gps_summaries(
-                    "max_dist_home",
-                    "radius",
-                    "diameter",
-                    "num_sig_places",
-                    "entropy",
-                    "total_flight_time",
-                    "av_flight_length",
-                    "sd_flight_length",
-                    "av_flight_duration",
-                    "sd_flight_duration",
-                    "total_pause_time",
-                    "av_pause_duration",
-                    "sd_pause_duration",
-                ]
-                + places_of_interest2
-                + places_of_interest3
-            )
 
-    if split_day_night:
-        summary_stats_df_daytime = summary_stats_df[::2].reset_index(
-            drop=True
-            )
-        summary_stats_df_nighttime = summary_stats_df[1::2].reset_index(
-            drop=True
+            summary_stats, log_tags = final_daily_prep(
+                obs_dur, obs_day, obs_night, time_at_home, dist_traveled,
+                max_dist_home, radius, diameter, num_sig, entropy,
+                total_flight_time, total_pause_time, flight_pause_stats,
+                all_place_times, all_place_times_adjusted, summary_stats,
+                log_tags, log_tags_temp, datetime_list, places_of_interest,
+                parameters, pcr, pcr_stratified, i
             )
 
-        summary_stats_df2 = pd.concat(
-            [
-                summary_stats_df_daytime,
-                summary_stats_df_nighttime.iloc[:, 3:],
-            ],
-            axis=1,
-        )
-        summary_stats_df2.columns = (
-            list(summary_stats_df.columns)[:3]
-            + [
-                f"{cname}_daytime"
-                for cname in list(summary_stats_df.columns)[3:]
-            ]
-            + [
-                f"{cname}_nighttime"
-                for cname in list(summary_stats_df.columns)[3:]
-            ]
-        )
-        summary_stats_df2 = summary_stats_df2.drop(
-            [
-                "obs_day_daytime",
-                "obs_night_daytime",
-                "obs_day_nighttime",
-                "obs_night_nighttime",
-            ],
-            axis=1,
-        )
-        summary_stats_df2.insert(
-            3,
-            "obs_duration",
-            summary_stats_df2["obs_duration_daytime"]
-            + summary_stats_df2["obs_duration_nighttime"],
-        )
-    else:
-        summary_stats_df2 = summary_stats_df
+    summary_stats_df2, log_tags = format_summary_stats(
+        summary_stats, log_tags, frequency, parameters, places_of_interest
+    )
 
     return summary_stats_df2, log_tags
 
 
+def split_day_night_cols(summary_stats_df: pd.DataFrame) -> pd.DataFrame:
+    """This function splits the summary statistics dataframe
+    into daytime and nighttime columns.
+
+    Args:
+        summary_stats_df: pandas dataframe with summary statistics
+    Returns:
+        summary_stats_df2: pandas dataframe with summary statistics
```

Comment:
No need to mention returned variable name since it is not a part of the API.
https://google.github.io/styleguide/pyguide.html#doc-function-returns